.github/workflows/formatter.yml

```diff
@@ -0,0 +1,54 @@
+name: Format
+
+on:
+  pull_request:
+    paths: ['**.cs']
+  push:
+    branches-ignore: [master]
```

Comment:
I think that running on a `pull_request` is enough. As far as I know, it will run every time when the PR branch is updated. So there is no need to use the `push` trigger:
 ```suggestion
    
```

Reply:
Hmm, I’ll test this out.

Reply:
You're right. I always assumed that the `pull_request` trigger only runs when a new PR is created.